src/main/java/org/cloudfoundry/promregator/PromregatorApplication.java

```diff
@@ -167,7 +167,7 @@ public InternalMetrics internalMetrics() {
 	
 	@Value("${promregator.endpoint.threads:#{null}}")
 	@Deprecated
-	/**
+	/*
```

Comment:
I am a little unsure whether removing the javadoc aspect here is the right thing. In Eclipse, it's no longer shown what shall be used instead...

Reply:
Its either a javadoc or a comment.. If its a javadoc, it should be fully javadocced .. if its just a comment, we treat it as a comment. My understanding of the issue above is that its currently half javadoc, half comment.

so either
/*  
xxxx
*/

or 

/**
Javadockystuff
**/